Changed output of ResourceGraph cmdlet to a custom class (#14789)
Azure/azure-powershell: src/ResourceGraph/ResourceGraph/ChangeLog.md

        - Additional information about change #1
-->
## Upcoming Release

> * Output has changed to a PSResourceGraphResponse class. Previous output is now under Data property.

Reviewer: ```suggestion
* Changed output of `Search-AzGraph` to PSResourceGraphResponse which wrapped previous output under Data property.
```
